Backend/src/main/java/com/onbank/starter/InitMockDB.java

```diff
@@ -0,0 +1,154 @@
+package com.onbank.starter;
+
+import com.onbank.api.model.Transfer;
+import com.onbank.api.model.enums.OperationType;
+import com.onbank.api.model.enums.TransferState;
+import com.onbank.api.repository.TransferRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Component
+public class InitMockDB {
+
+    private static final String MARIUSZ_KOWALSKI = "Mariusz Kowalski";
```

Comment:
Nazwy też proponuje zmienić no bo teraz to żadna różnica :D, już coś w rodzaju TEST_NAME_1 będzie lepsze